pom.xml

```diff
@@ -238,6 +238,8 @@
     <error-prone-support.version>0.14.0</error-prone-support.version>
     <doxia.version>1.12.0</doxia.version>
     <error-prone.configuration-args>
+      <!-- Prefer `.isNull()` over `.isEqualTo(null)`. -->
```

Comment:
Please remove this comment, explanation what rule does is available in web.
ERROR is definition that we are serious to follow it

Reply:
Done.